spikeinterface/core/__init__.py

```diff
@@ -28,6 +28,11 @@
 from .channelsaggregationrecording import ChannelsAggregationRecording, aggregate_channels
 from .unitsaggregationsorting import UnitsAggregationSorting, aggregate_units
 
+# utility sorters
+from .mergesorting import MergeSorting
+from .splitsortingunit import SplitSortingUnit
```

Comment:
```suggestion
from .mergesorting import MergeUnitSorting
from .splitsortingunit import SplitUnitSorting
```

IMO we can use these names for consistency. What do you think?

Reply:
maybe... MergeSortingUnit and SplitSortingUnit, sound clearest for me. Or even something like SplitUnit and MergeUnits. All of them works are better than the current ones (I started playing with graphs and went lazy with the details). Perhaps @samuelgarcia has a strong preference for any of them?